app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardBody.js

```diff
@@ -23,6 +24,11 @@ const MappingWizardBody = props => {
   const mappingWizardClustersStepContainer = componentRegistry.markup(
     'MappingWizardClustersStepContainer'
   );
+  // This may not be required for Redux form
```

Comment:
More commented out code to remove before we merge

Reply:
Addressed in d3da630